src/LiquidationRouter.sol

```diff
@@ -30,6 +30,7 @@ contract LiquidationRouter {
     uint256 _amountIn,
     uint256 _amountOutMin
   ) external returns (uint256) {
+    require(_liquidationPairFactory.deployedPairs(_liquidationPair), "LR/LP-not-from-LPF");
     IERC20(_liquidationPair.tokenIn()).safeTransferFrom(
       _account,
```

Comment:
It should be `msg.sender` here.
Refer to the following Discord chat: https://discord.com/channels/629836507280048129/636334751069896714/1078074087097774170
```suggestion
      msg.sender,
```